modules/15_climate/ar6/declarations.gms

```diff
@@ -0,0 +1,47 @@
+*** |  (C) 2006-2023 Potsdam Institute for Climate Impact Research (PIK)
```

Comment:
I would actually be in favor of having MAGICC7 somewhere in the module name, because the AR6 used different climate models. Why not MAGICC7_AR6?

Reply:
That's fine by me! The other `15_climate` module option is `magicc`, so `magicc7_ar6`  would match 👍